src/Esfa.Vacancy.Api.Core/ActionFilters/ProviderAuthorisationFilter.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Web.Http.Controllers;
+using System.Web.Http.Filters;
+using Esfa.Vacancy.Api.Core.Extensions;
+using Esfa.Vacancy.Application.Exceptions;
+
+namespace Esfa.Vacancy.Api.Core.ActionFilters
+{
+    public class ProviderAuthorisationFilter : ActionFilterAttribute
```

Comment:
Not sure about this filter. It reads a header just to add a header which we still have to inspect in the orchestrator.  Could we just add an Extension Method in HttpRequestExtensions.cs to return the UKPRN. We can then just pass the UKPRN straight into the orchestrator.